saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/SpatialRoleLabeling/README.md

```diff
@@ -0,0 +1,111 @@
+# Spatial Role Labeling
```

Comment:
Could you add the link+description to the examples readme file? 
https://github.com/IllinoisCogComp/saul/blob/master/saul-examples/README.md